### PR TITLE
Posthog Toolbar

### PR DESCRIPTION
**Pull request by mariusandra**
This adds supports for launching the new posthog toolbar.js when `toolbarVersion` starts with `toolbar` (and in the future `toolbar.v1` or so). Also adds `jsURL` for enabling HMR support.

I'd release 1.1.0 once this is in.

Also, I noticed that posthog-js is quite noisy. Printing error messages I get, yet some of these `console.log`s here are for sure leftover debug statements:

image

I'd remove them before pushing out 1.1.0. @timgl any opinion?

### CODE REVIEW

File: src/autocapture.js [hunk 1]
```diff
@@ -221,6 +221,14 @@ var autocapture = {
 
         if (!this._maybeLoadEditor(instance)) { // don't autocapture actions when the editor is enabled
             var parseDecideResponse = _.bind(function(response) {
+                if(response['isAuthenticated'] && response['toolbarVersion'].indexOf('toolbar') === 0) {
+                    this._loadEditor(instance, {
+                        apiURL: instance.get_config('api_host'),
+                        jsURL: response['jsURL'] || instance.get_config('api_host'),
```
Comment (timgl) [inline src/autocapture.js:227]: What's the difference between jsURL and api_host/apiURL?

Comment (mariusandra) [reply src/autocapture.js:227]: when `settings.DEBUG`, `jsURL` will be `http://localhost:8234` - the HMR url

Comment (timgl) [reply src/autocapture.js:227]: Okay makes sense

**Comment by mariusandra** [comment]
Thanks, merged and posthog-js v 1.1.0 pushed out. We should communicate the necessity to upgrade to this version when announcing the toolbar... for those who have posthog-js installed via npm/yarn... and not via the snippet.

**Pull request merged**